packages/react-core/src/demos/DataListDemo.md

```diff
@@ -418,3 +426,348 @@ class ExpandableDataList extends React.Component {
+                      <DataListCell isFilled={false} key="buttons4">
+                        <Flex direction={{ default: 'column' }}>
+                          <FlexItem>
+                            <Text component={TextVariants.small}>patternfly</Text>
+                          </FlexItem>
+                          <FlexItem>
+                            <Text component={TextVariants.small}>
+                              Working repo for PatternFly 4{' '}
+                              <Button variant="link" isInline>
+                                https://pf4.patternfly.org/
+                              </Button>
+                            </Text>
+                          </FlexItem>
+                          <FlexItem>
+                            <Flex>
+                              <FlexItem>
+                                <CodeBranchIcon />
+                                \xa0 10
+                              </FlexItem>
+                              <FlexItem>
+                                \xa0 <CodeIcon /> \xa0 4
+                              </FlexItem>
+                              <FlexItem>
+                                \xa0 <CubeIcon /> \xa0 5
+                              </FlexItem>
+                              <FlexItem> \xa0 \xa0 Updated 2 days ago</FlexItem>
+                            </Flex>
+                          </FlexItem>
+                        </Flex>
+                      </DataListCell>,
+                      <DataListCell isFilled={false} alignRight key="secondary content align">
+                        <Flex>
+                          <FlexItem>
+                            <Button variant="secondary">Action</Button>
+                          </FlexItem>
+                          <FlexItem>
+                            <Button variant="link" isInline>
+                              Link
+                            </Button>
+                          </FlexItem>
+                        </Flex>
+                      </DataListCell>
+                    ]}
+                  />
+                </DataListItemRow>
+              </DataListItem>
+              <DataListItem aria-labelledby="Demo-item5">
+                <DataListItemRow>
+                  <DataListItemCells
+                    dataListCells={[
+                      <DataListCell isFilled={false} key="buttons5">
+                        <Flex direction={{ default: 'column' }}>
+                          <FlexItem>
+                            <Text component={TextVariants.small}>patternfly-elements</Text>
+                          </FlexItem>
+                          <FlexItem>
+                            <Text component={TextVariants.small}>PatternFly elements</Text>
+                          </FlexItem>
+                          <FlexItem>
+                            <Flex>
+                              <FlexItem>
+                                <CodeBranchIcon /> \xa0 5
```

Comment:
rather than use all these unicode `\xa0` characters, you can add `spaceItems={{ default: "spaceItemsSm" }}` to the wrapping `Flex` component.

Reply:
I applied that comment!